dora/tests/src/test/java/alluxio/client/cli/fs/command/HeadCommandIntegrationTest.java

```diff
@@ -18,11 +18,14 @@
 import alluxio.util.io.BufferUtils;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Tests for head command.
  */
+// TODO(JiamingMai): <TEST_FIX> Fix this test or remove it if it is deprecated
+@Ignore("Master no longer manages metadata")
```

Comment:
Honestly I don't know how to handle this kind of work. @Xenorith Is this related to your work on reorganizing commands? I figure we should find a way to keep the UTs, meaning it is fine to @Ignore them now but we should remember to dig them back up once the command line reorg is done.